subsys/bluetooth/controller/ll_sw/ll.c

```diff
@@ -26,6 +26,10 @@
 #include "hal/radio.h"
 #include "hal/debug.h"
 
+#if defined(CONFIG_SOC_FAMILY_NRF5)
+#include "hal/nrf5/ticker.h"
```

Comment:
Same comment at in flash driver (i.e. consider including the public header, instead)

Reply:
Same reply as in flash driver comment.